.templates/new-module/CHANGELOG.md

```diff
@@ -20,4 +20,4 @@
 **[⬆ back to top](#content)**
 
 
-# };
\ No newline at end of file
+# };
```

Comment:
Just a note here. We should add `insert_final_newline = false` to `[*.md]` inside the `.editorconfig` to avoid changes like this. Wanna do that?